src/test/java/com/virtuslab/archunit/UIThreadUnsafeMethodInvocationsTestSuite.java

```diff
@@ -153,6 +153,9 @@ public void check(JavaMethod method, ConditionEvents events) {
       "java.io.File.getAbsolutePath()",
       "java.io.File.isFile()",
       "java.io.File.toString()",
+      "java.io.BufferedOutputStream.<init>(java.io.OutputStream)",
```

Comment:
Since these methods are already whitelisted here, there's no need to include them in `@IgnoreUIThreadUnsafeCalls` at specific methods (e.g. `com.virtuslab.gitmachete.frontend.file.MacheteFileWriter#writeBranchLayout`).

Although... TBH I'm not sure whether methods like `close()` are **really** safe to call on UI thread in general case. Depending on the I/O stream implementation, `close()` might be the place where e.g. the buffer is actually flushed to the disk. So I'd rather remove these global whitelisting, and leave `@IgnoreUIThreadUnsafeCalls` at specific methods.